src/panels/lovelace/hui-root.ts

```diff
@@ -513,6 +513,11 @@ class HUIRoot extends LitElement {
     });
   }
 
+  public connectedCallback(): void {
+    super.connectedCallback();
+    window.addEventListener("scroll", this._handleWindowScroll);
```

Comment:
should it set `passive: true` like the first time?

Reply:
Yes you are right :wink:. I've amended my PR with a3c6476f672ce062a2557035ee9b7f5ece3a348c. Functionally it shouldn't matter, but it [improves performance](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#using_passive_listeners).